### DISCUSSION TITLE
Redirect sound from out to blackhole using Boom2 mac app

### DISCUSSION THREAD

**Discussion by vlad-cotfas** [ExistentialAudio/BlackHole#224] [General]
Mac OSX Catalina 10.15.7 (19H15)
BlackHole BlackHole2ch.v0.2.7.pkg
Boom2 mac app 1.6.12

I use the Boom2Device for output, the boom2 takes the in default sound and apply sound effects, how I can redirect the in sound (not the boom2 out device) to blackhole driver so other app can use BH as input device?

Thank you

**Comment by devinroth** [comment]
That sounds like a question for Boom. 

I don’t know the answer but I’ll leave this up. Perhaps someone else knows.

**Comment by vlad-cotfas** [comment]
Not 100% necessary for boom2.

How I can take the output speakers (audio out) and redirect it to BH in?

**Comment by devinroth** [comment]
You need to output to BlackHole and then output BlackHole to the speakers.

That feature isn’t built into BlackHole but if you search around in the Github issues some people have figured out ways to do it.

**Comment by vlad-cotfas** [comment]
I do not want to use other output device (eg. BH) as I need to use Boom2Driver as output otherwise I loose the boom2 functionality. Ideally would be to redirect default mac + boom2driver(as some of the times boom2 stop working and mac selects default output) + external speakers + external headphones (I think those 2 would be already in default out, I think).

I`ve searched for a lot of free options on github but nothing found, I found something that would work 'Proxy Audio Driver' but I can not use the mic IN as well in the driver OUT.
